website/src/app/room/room.component.ts

```diff
@@ -95,13 +93,31 @@ export class RoomComponent implements OnInit, OnDestroy, AfterViewInit {
         this.alertsDataSource = new MatTableDataSource([...this.issue.alerts.values()]);
         this.alertsDataSource.sort = this.sort;
       }
+      this.api.getIssueType().subscribe(info =>{
```

Comment:
Let's move this part out of update, we could maybe put it in the constructor. Since the issueType data isn't going to change very often we don't need to get it every 10 seconds. We can just get it once when the page loads and then use that same data throughout the other updates.